### DISCUSSION TITLE
Can't launch because of attribute error - 'Options' object has no attribute 'ui_reorder'. Did you mean 'reorder'?

### DISCUSSION THREAD

**Discussion by offroadguy56** [Gerschel/sd_web_ui_preset_utils#26] [General]
I am getting an error when launching the webui. See below. I unfortunately don't program and am new to Git. Please correct me, assist me, or point me in the right direction for this error. If you need anymore info let me know. 

Thanks

![image](https://user-images.githubusercontent.com/6194572/210450626-12112854-fe06-40fc-aad0-41babd1a8fd6.png)

**Comment by Gerschel** [comment]
stable-diffusion-webui had an update where a user can reposition some rows.  
See if updating it solves this.  
Let me now if it doesn't.  

This error is saying that the setting it reads, does not exist. It's a new setting that was added 15 hours ago.  
On an aside tip. A lot of the recent updates broke many things, so check updates for your other extensions. Apply them. Shut it back down. Restart (so it can install some new dependencies). You might  have to restart again, I did. Force reset browser cache with ctrl+f5 after it loads in the browser.

**Reply by offroadguy56** [reply]
Thanks for the quick response. I am out of the house right now. I'll work on your suggestion if I have time tonight. I might not be able to get around to fixing this until thursday/friday.

**Reply by Gerschel** [reply]
If you need the non-updated version of the script, the prior one, you can do a checkout.  A checkout replaces the files with the ones from the given address or branch or source. Then when you're ready, you can checkout back to the master branch (a checkout from history acts like a new branch).

```
git checkout 988be227277ed9d88583b5624d2758f3bab61b23
```  

Then to checkout back to the master branch.  

```
git checkout master
```

**Reply by offroadguy56** [reply]
I had just enough time to futz around. I tried to `git pull` the webui like it recommended but it ended up aborting, something about committing changes. I just ended up downloading the zip and replacing all files. It appears to be working now. I am all out of time and will have to properly test the extension later. But it has definitely shown up and am super excited to put it to use. 
Thank you!

**Reply by mocmocmoc** [reply]
> stable-diffusion-webui had an update where a user can reposition some rows. See if updating it solves this. Let me now if it doesn't.

possible to move it to top at Quick Settings or bottom of the Image Gallery?

**Reply by Gerschel** [reply]
> > stable-diffusion-webui had an update where a user can reposition some rows. See if updating it solves this. Let me now if it doesn't.
> 
> possible to move it to top at Quick Settings or bottom of the Image Gallery?

Moved this to a new issue. So it can be discussed there. I would like others feedback as well, for placement. https://github.com/Gerschel/sd_web_ui_preset_utils/issues/27

**Comment by Gerschel** [comment]
https://github.com/Gerschel/sd_web_ui_preset_utils/discussions/29#discussioncomment-4704061
